pkg/apim/APIMUtility.go

```diff
@@ -51,3 +66,19 @@ func (apim *Bundle) IsPayloadJSON() bool {
 	var js json.RawMessage
 	return json.Unmarshal([]byte(apim.Payload), &js) == nil
 }
+
+func (odataFilters *OdataParameters) MakeOdataQuery() (string, error) {
```

Comment:
This function is way to complicated, there must be a simpler way. Maybe by not using the `url` package?

Reply:
its already simplified using qs library

Reply:
yes, done.